ipfs/evolution.md

```diff
@@ -1 +1,143 @@
-# IPFS - Evolution
\ No newline at end of file
+# IPFS - Evolution
+
+The IPFS project was born in 2013 by Juan Benet,  Juan  known Git's data-linking structure is useful across many data types–not just code. Juan recognized that combining this concept with the P2P file-sharing structure of BitTorrent would be powerful: functional, secure information sharing without centralized barriers. This could transform the world far beyond scientific datasets.
```

Comment:
I dont understand what goes after the comma "Juan known Git's.." is the are a point missing? Do you mean "Juan knew"